android/app/src/main/java/com/breez/client/job/JobManager.java

```diff
@@ -31,6 +32,6 @@ private void startSyncJob(){
                                 .build())
                 .build();
 
-        WorkManager.getInstance().enqueue(oneTime);        
+        WorkManager.getInstance().enqueueUniqueWork(UNIQUE_WORK_NAME, ExistingWorkPolicy.REPLACE, oneTime);        
```

Comment:
@JssDWt did you consider using https://developer.android.com/reference/androidx/work/ExistingWorkPolicy#KEEP ? It looks like the difference is that it won't cancel the pending job and just let it finish.

Reply:
I did not consider that, that looks better indeed

Reply:
Testing this out it seems the job is not executed when I use `ExistingWorkPolicy.KEEP`